anax-in-container/horizon-container

```diff
@@ -378,6 +378,7 @@ stop() {
 
 	# Stop the anax container
 	echo "Unregistering the node, then stopping/removing the horizon container (this may take a minute)..."
+	/usr/bin/hzn unregister -f
```

Comment:
/usr/bin/hzn is not a path found in MacOS... I think this can just be `hzn unregister -f` or determine if you are Mac or Linux
```
larson@Dougs-MacBook-Pro-2 ~ % which hzn
/usr/local/bin/hzn
```